raiden/routing.py

```diff
@@ -247,9 +247,9 @@ def get_best_routes_pfs(
             value=amount,
         )
     except ServiceRequestFailed as e:
-        log_message = e.args[0]
+        log_message = ("PFS: " + e.args[0]) if e.args[0] else None
```

Comment:
```suggestion
        log_message = ("PFS: " + e.args[0]) if e.args else None
```

To prevent the index error

Reply:
That is what I tried first, but it fails when `e.args` is `[None]`, because it tries to do `"PFS: " + None`. Tuples also don't have a `get` and I did not really was hesitant to add another `if` layer.

Reply:
Ah i see so the list will have args but we're checking the values.